components/SpeakersSection.js

```diff
@@ -108,6 +111,23 @@ function Speaker ({ name, title, photo }) {
           letter-spacing: 0.05em;
           opacity: 0.5;
         }
+
+        .speaker-info {
+          max-width: 270px;
+          margin: auto auto;
+          height: 270px;
+        }
+
+        .description {
+          margin-top: 8px;
+          font-style: italic;
+        }
+
+        @media (max-width: 768px) {
+          .speaker-info {
+            height: inherit;
```

Comment:
I think `height: inherit;` isn’t what you meant to do.

`inherit` means “take the value from the parent element,” in this case, it is `auto`.

Therefore, I suggest do one of these instead:

```css
height: auto;
```

or

```css
height: initial;
```